build/build.rb

```diff
@@ -89,6 +89,10 @@ def verify_run(output)
     file_name = "#{name}-#{target}-#{build_label}-#{timestamp}-#{git_checkout.commit_sha}.#{FILE_TYPE[imgfac_target]}"
     destination = DESTINATION_DIRECTORY.join(file_name)
     $log.info `mv  #{source} #{destination}`
+    $log.info "Generating image checksums"
```

Comment:
@JPrause Seems perhaps that I'm not being clear with what I'm suggesting.
Let me try a code segment. If this still isn't clear let's chat.

I'm suggesting you consider changing lines 93-95 from:
    $log.info `cd #{DESTINATION_DIRECTORY}/; /usr/bin/sha256sum * >> SHA256SUM`
    $log.info `cd #{DESTINATION_DIRECTORY}/; /usr/bin/gpg --batch --no-tty --passphrase-file /root/.gnupg/pass -b SHA256SUM`
    $log.info `cd #{DESTINATION_DIRECTORY}/; cp /root/.gnupg/cfme_public.key . `

To:
Dir.chdir(DESTINATION_DIRECTORY) do
  $log.info `/usr/bin/sha256sum * >> SHA256SUM`
  $log.info `/usr/bin/gpg --batch --no-tty --passphrase-file /root/.gnupg/pass -b SHA256SUM`
  FileUtils.cp("/root/.gnupg/cfme_public.key", DESTINATION_DIRECTORY)
end 


I think that will work for you and allow you to avoid issuing the "cd" before each of the 3 commands.